### PR TITLE
Improved Phalcon\Config\Adapter\Ini

### PR DESCRIPTION
**Pull request by dred86**
I've modified Ini.zep

### CODE REVIEW

**Comment by mruz** [comment]
Add `strtolower`
```zep
if ini === "true" || ini === "yes" || strtolower(ini) === "on"{
...
```
and fix a test https://github.com/phalcon/cphalcon/issues/11863#issuecomment-225808800

File: phalcon/config/adapter/ini.zep [hunk 1]
```diff
@@ -118,4 +120,47 @@ class Ini extends Config
...
 	}
+	
+    /**
+     * @link https://github.com/ice/framework/blob/dev/ice/config/ini.zep#L46
+     * We have to cast values manually because parse_ini_file() has a poor implementation.
+     *
+     * @param mixed ini The array casted by `parse_ini_file`
+     * @return mixed
+     */
+    private function _cast(ini)
```
Comment (sergeyklay) [inline phalcon/config/adapter/ini.zep]: 1. `-> var` and remome `@return`
2. after a few months it is likely the `@link` will not be valid (at least line number)

**Comment by sergeyklay** [comment]
@dred86

**Comment by dred86** [comment]
Sorry I do not know yet how to use git

**Comment by dred86** [comment]
https://github.com/dred86/cphalcon/blob/config-ini/phalcon/config/adapter/ini.zep

**Comment by sergeyklay** [comment]
We don't add changes directly to stable branches or `master`. You have to send this PR to the `2.1.x` branch. Also note to the comments

**Pull request closed without merging**